sysstat.service.in

```diff
@@ -2,20 +2,27 @@
 # (C) 2012 Peter Schiffer (pschiffe <at> redhat.com)
 #
 # @PACKAGE_NAME@-@PACKAGE_VERSION@ systemd unit file:
-#	 Insert a dummy record in current daily data file.
-#	 This indicates that the counters have restarted from 0.
+#     1. Insert a dummy record in current daily data file when starting/stopping
+#        the unit. This indicates that the counters have restarted from 0.
+#     2. This unit is automatically stopped/started on suspend/resume,
+#        see /usr/lib/systemd/system-sleep/sysstat-sleep.sh.
+#     3. Starting this unit also starts the timer unit for periodic accounting.
 
 [Unit]
-Description=Resets System Activity Logs
+Description=System Activity Accounting
+# Start the periodic accounting process
+Requires=sysstat-collect.timer
+# but make sure we finish first, because the accounting process locks the datafile
+Before=sysstat-collect.timer sysstat-collect.service
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
 User=@CRON_OWNER@
-ExecStart=@SA_LIB_DIR@/sa1 --boot
+ExecStart=@SA_LIB_DIR@/sa1 --boot -C STARTED
```

Comment:
@sysstat I was thinking, can't a reboot be detected automatically? Exactly because some counters are monotonic, that should be possible. And this init.d service script that just inserts a comment has always struck me as strange, maybe out-of-place.

On the other hand suspend and wake-up can't be detected because counters are not reset. Well, a jump in time is the only evidence, but sometimes it happens for other reasons.

Anyway I hope I'll get some time to update this pull request until next week!